src/components/app.js

```diff
@@ -71,6 +71,15 @@ export default class App extends Component {
         <RepositoryList
           items={state.items}
         />
+        <div className="footer">
+           <ul>
+             <li id="footer-title"><a href="https://github.com/ivanseed/gitstats"><span>Git<b>Stats</b></span></a></li>
+             <li>
+               <span>Special thanks to all contributors that made this project possible!</span>
+               <span><b><a href="https://github.com/ivanseed">ivanseed</a></b>, <b><a href="https://github.com/haroon-sheikh">haroon-sheikh</a></b>, <b><a href="https://github.com/athyk">athyk</a></b></span>
```

Comment:
Is it possible to generate this from a GitHub API request? 
https://developer.github.com/v3/repos/#list-contributors

You would probably want to put all of this into its own separate, stateless, component.